docs/source/pipelines.mdx

```diff
@@ -0,0 +1,215 @@
+<!--Copyright 2022 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# Optimum pipelines for inference
+
+The [`optimum_pipeline`] makes it simple to use models from the [Model Hub](https://huggingface.co/models) for accelerated inference on a variety of tasks such as text classification.
+Even if you don't have experience with a specific modality or understand the code powering the models, you can still use them with the [`optimum_pipeline`]! This tutorial will teach you to:
+
+<Tip>
+
+You can also use the `transformers.pipeline` and provide your `OptimumModel`.
+
+</Tip>
+
+Currenlty supported tasks are:
+
+**Onnxruntime**
+
+* `feature-extraction`
+* `text-classification`
+* `token-classification`
+* `question-answering`
+* `zero-shot-classification`
+* `text-generation`
+
+## Optimum pipeline usage
+
+While each task has an associated [`optimum_pipeline`], it is simpler to use the general [`pipeline`] abstraction which contains all the specific task pipelines. 
+The [`optimum_pipeline`] automatically loads a default model and tokenizer capable of inference for your task. 
+
+1. Start by creating a [`optimum_pipeline`] and specify an inference task:
+
+```py
+>>> from optimum import optimum_pipeline
+
+>>> classifier = optimum_pipeline(task="text-classification", accelerator="ort")
+
+```
+
+2. Pass your input text to the [`optimum_pipeline`]:
+
+```python
+>>> classifier("I like you. I love you.")
+```
+
+### Use a vanilla transformers model and convert
```

Comment:
Not sure if you meant "convert to generic IR" so feel free to drop the ONNX part
```suggestion
### Use a vanilla Transformers model and converting to ONNX
```